rotkehlchen/tests/api/test_liquity.py

```diff
@@ -252,7 +254,10 @@ def test_staking_stats(rotkehlchen_api_server, ethereum_accounts):
     assert len(global_stats['staking_gains']) == 3
     assert len(global_stats['stability_pool_gains']) == 2
     assert FVal(global_stats['stability_pool_gains'][0]['amount']).is_close(FVal(address_0_data['stability_pool_gains'][0]['amount']) + FVal(address_1_data['stability_pool_gains'][0]['amount']), max_diff='1e-8')  # noqa: E501
-    assert address_0_data['stability_pool_gains'][0]['amount'] == '1.7820064710306824'
+    assert math.isclose(
```

Comment:
huh that's interesting. Any idea why this is different for windows only?

What you should do instead btw is use the `Fval` functionality we got which you can see in action in the line right above as I think `address_0_data['stability_pool_gains'][0]['amount']` is already an `FVal`

Reply:
> huh that's interesting. Any idea why this is different for windows only?

Not sure, but it's off by `2e-16` which I would put down to floating point precision differences between OSs

> What you should do instead btw is use the Fval functionality we got which you can see in action in the line right above as I think address_0_data['stability_pool_gains'][0]['amount'] is already an FVal

Oh cool, I didn't see that. Neat, thanks!